src/Flexbox.jsx

```diff
@@ -1,39 +1,38 @@
-import React, { PropTypes } from 'react';
+import PropTypes from 'prop-types';
+import React from 'react';
 import styled from 'styled-components';
 
-const { string, oneOf, node, number, bool, object } = PropTypes;
```

Comment:
I would keep this approach instead of calling `PropTypes.` everywhere. I suppose you have your reasons to move towards using `PropTypes.`, want to hear your thoughts about that!